README.rst

```diff
@@ -48,6 +48,20 @@ Each time you call ``generate_jwt`` this will find the latest active key file (e
     signer = atlassian_jwt_auth.create_signer_from_file_private_key_repository('issuer', '/opt/jwtprivatekeys')
     a_jwt = signer.generate_jwt('audience')
 
+To create a JWT using a data uri
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import atlassian_jwt_auth
+    from atlassian_jwt_auth.key import DataUriPrivateKeyRetriever
+
+    pem_id, private_key_pem = DataUriPrivateKeyRetriever('Your base64 encoded data uri').load('issuer/key')
```

Comment:
@itsrifat - the `.load` method takes an `issuer` and not a `key identifier` - so arguably this should be `DataUriPrivateKeyRetriever('Your base64 encoded data uri').load('issuer')`.